### DISCUSSION TITLE
I have not received the cultivation this weekend

### DISCUSSION THREAD

**Discussion by MPCMINER** [Chia-Network/chia-blockchain#7042] [Support]
Dear friends, I already have more than 1700 plots, this weekend you have stopped the payments while you were cleaning duplicate plots, I had to charge 0.06 ,,,, chias plus those cultivated in the weekend, yesterday that you made the payment I have not received the payment and above in my pool I appeared to collect pending 0.45 ,,,,, what happened to my chias ?????

Thank you

**Comment by loppefaaret** [comment]
This is the wrong place
you need to contact whatever pool that you are having trouble with!
Chia Network does not operate pools, we have yet to even release pooling protocol!

**Comment by BrandtH22** [comment]
Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
